docs/TextRegion.md

```diff
@@ -44,12 +44,12 @@ The horizontal start position will be either at the current x position, if that
 In both horizontal and vertical positioning, regions with multiple columns may follow additional rules and restrictions.
 
 
-### Interaction between Regions ###
+### Interaction Between Regions ###
```

Comment:
Capitalizing all words is not really the norm in English 😅

Reply:
It actually is in titles and headings (except for articles, prepositions, and similar small words).